.gitattributes

```diff
@@ -1,2 +1,9 @@
-# Auto detect text files and perform LF normalization
-* text=auto
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
+# Mark the database schema as having been generated.
```

Comment:
- [ ] Kindly, view the review comment in the frontend [pr](https://github.com/BHObserver/Rent-a-Car-frontend/pull/22). 👍